dss/api/files.py

```diff
@@ -108,6 +109,39 @@ def get_helper(uuid: str, replica: Replica, version: str=None):
     return response
 
 
+def _verify_checkout(
+        replica: Replica, token: typing.Optional[str], file_metadata: dict, blob_path: str,
+) -> typing.Tuple[str, bool]:
+    decoded_token: dict
+    if token is None:
+        execution_id = start_file_checkout(blob_path, replica)
+        start_time = time.time()
+        attempts = 0
+
+        decoded_token = {
+            CheckoutTokenKeys.EXECUTION_ID: execution_id,
+            CheckoutTokenKeys.START_TIME: start_time,
+            CheckoutTokenKeys.ATTEMPTS: attempts
+        }
+    else:
+        try:
+            decoded_token = json.loads(token)
+            decoded_token[CheckoutTokenKeys.ATTEMPTS] += 1
+        except Exception as ex:
```

Comment:
All exceptions? Why not just `(KeyError, ValueError)`?

Reply:
Good idea, will fix.